src/Scene.cpp

```diff
@@ -2,8 +2,10 @@
 #include "Scene.h"
 #include "Camera.h"
 
-Scene::Scene( Camera* cam ):
-    camera( cam ) {
+Scene::Scene( Shader* prog, Shader* progOutline, Camera* cam ):
```

Comment:
You took the shaders out from the scene in a previous PR. Think this is from  wrong merge.